README.md

```diff
@@ -107,6 +107,11 @@ Disable the following inspections:
 - ``Java | Performance | Call to 'Arrays.asList()' with too few arguments``,
 - ``Java | Abstraction issues | 'Optional' used as field or parameter type``.
 
+Enable errorprone ([Error Prone Installation#IDEA](https://errorprone.info/docs/installation#intellij-idea)):
+- Install ``Error Prone Compiler`` plugin from marketplace,
+- In ``Java Compiler`` tab, select ``Javac with error-prone`` as the compiler,
+- Update ``Additional command line parameters`` with ``-XepExcludedPaths:.*/target/generated-(|test-)sources/.* -XepDisableAllChecks -Xep:MissingOverride:ERROR -Xep:ClassCanBeStatic:ERROR -Xep:ArrayToString:ERROR -Xep:GetClassOnClass:ERROR -Xep:EqualsIncompatibleType:ERROR``
```

Comment:
That duplicates pom.xml configuration and will go out of sync.
It is too easy to forget about README in a PR like https://github.com/prestosql/presto/pull/4269.

Maybe say something like 
```
Update ``Additional command line parameters`` with
``-XepExcludedPaths:.*/target/generated-(|test-)sources/.* -XepDisableAllChecks -Xep:MissingOverride:ERROR .....`` 
(for current recommended list of command line parameters, see the top level ``pom.xml``, the definition of the
``errorprone-compiler-presto`` profile.
```